cmd/juju/caas/add.go

```diff
@@ -214,6 +220,19 @@ func (c *AddCAASCommand) Init(args []string) (err error) {
 	if c.contextName != "" && c.clusterName != "" {
 		return errors.New("only specify one of cluster-name or context-name, not both")
 	}
+	
```

Comment:
Need a list of clouds for which region is mandatory and enforce that region must be specified for those. For the others, allow just cloud type

Reply:
fixed